### ISSUE TITLE
Multiple reports prevent players from responding or playing the game

### ISSUE THREAD

**Issue by BadgerCode** [BadgerCode/tttdamagelogs#7]
# Steps to reproduce multiple reports issue
_This only seems to happen some of the time._

1. New round starts
2. Player A kills player B. Player B reports player A.
3. Player A kills player C. Player C reports player A.
4. Player A dies and is automatically requested to answer their reports
5. Player A answers player B's report and submits their answer.

**Expected result**
Player A's response to player B's report is submitted.
Player A is requested to answer player C's report.

**Actual result**
Player A's response to player B's report is submitted.
Player B's report is still visible.
Player C's report cannot be selected (clicking the button does nothing).
Submit button is disabled.

----

Here is what the reported player sees:<br>
![Report menu](https://cdn.discordapp.com/attachments/597161425210703884/665563250804523078/unknown.png)

----

```
Yugen|137|STEAM_0:1:XXX

[ERROR] addons/tttdamagelogs-master/lua/damagelogs/client/rdm_manager.lua:123: bad argument #1 to 'WriteUInt' (number expected, got nil)
  1. WriteUInt - [C]:-1
   2. DoClick - addons/tttdamagelogs-master/lua/damagelogs/client/rdm_manager.lua:123
    3. unknown - lua/vgui/dlabel.lua:234

Yugen|137|STEAM_0:1:XXX
[ERROR] Warning! A net message (DL_SendAnswer) is already started! Discarding in favor of the new message! (pac_update_playerfilter)
  1. unknown - addons/pac3-master/lua/pac3/editor/client/wear.lua:39
```

When an admin tries to change it to waiting

```
[ERROR] lua/includes/extensions/player.lua:75: bad argument #1 to 'upper' (string expected, got nil)
  1. upper - [C]:-1
   2. GetBySteamID - lua/includes/extensions/player.lua:75
    3. DoClick - addons/tttdamagelogs-master/lua/damagelogs/client/tabs/rdm_manager.lua:725
     4. unknown - lua/vgui/dlabel.lua:234
```

---

This can also be caused by entering a report reason longer than 2000 characters (e.g. 113k characters)

This is mentioned in the following issues:
* https://github.com/Tommy228/tttdamagelogs/issues/392
* https://github.com/Tommy228/tttdamagelogs/issues/390
* https://github.com/Tommy228/tttdamagelogs/issues/368

**Comment by BadgerCode** [comment]
![preview](https://i.imgur.com/r9GErGM.png)

**Comment by BadgerCode** [comment]
I have worked out the actual steps to reproduce:

1. Get reported
2. Open the report answer interface (either by dying, having it open automatically, typing !respond, ...)
3. Get reported by someone else


This triggers a different flow in `lua\damagelogs\client\rdm_manager.lua`, which fails to add the new report to the interface, but prevents the player from closing the interface.

**Comment by BadgerCode** [comment]
Fixed in #11 
Will be released as a part of #9
